IDESettings/.clang-format

```diff
@@ -0,0 +1,116 @@
+---
+Language: Cpp
+AccessModifierOffset: -4
+AlignAfterOpenBracket: DontAlign
```

Comment:
AlwaysBreak?

Reply:
I think it's not fully doing what we do but would be a step towards it (see PR description on it and `BinPack*`)